addons/website_event_meet/controllers/main.py

```diff
@@ -0,0 +1,121 @@
+# -*- coding: utf-8 -*-
+# Part of Odoo. See LICENSE file for full copyright and licensing details.
+
+import logging
+from werkzeug.exceptions import Forbidden
+from werkzeug.utils import redirect
+
+from odoo import http
+from odoo.http import request
+from odoo.addons.http_routing.models.ir_http import slug
+
+
+_logger = logging.getLogger(__name__)
+
+
+class WebsiteEventMeetController(http.Controller):
+    @http.route(
+        ["/event/<model('event.event'):event>/meeting_rooms"],
+        type="http",
+        auth="public",
+        website=True,
+        sitemap=True,
+    )
+    def event_meeting_rooms(self, event, lang=None, open_room=None):
+        if not event.can_access_from_current_website():
+            raise Forbidden()
+
+        meeting_rooms = event.sudo().meeting_room_ids
+
+        if lang is not None:
+            lang = request.env["res.lang"].browse(int(lang))
+            meeting_rooms = meeting_rooms.filtered(lambda m: m.lang_id == lang)
+
+        values = {
+            "event": event.sudo(),
+            "meeting_rooms": meeting_rooms,
+            "current_lang": lang,
+            "available_languages": event.sudo().meeting_room_ids.mapped("lang_id"),
+            "open_room": int(open_room) if open_room else None,
+            "is_event_manager": request.env.user.has_group("event.group_event_manager"),
+            "default_lang_code": request.env.user.lang,
+        }
+
+        return request.render("website_event_meet.template_meeting_rooms", values)
+
+    @http.route(["/event/create_meeting_room"], type="http", auth="public", methods=["POST"])
+    def create_meeting_room(self):
+        name = http.request.params.get("name")
+        summary = http.request.params.get("summary")
+        target_audience = http.request.params.get("audience")
+        lang = http.request.params.get("lang")
+        max_capacity = http.request.params.get("capacity")
+        event_id = int(http.request.params.get("event"))
```

Comment:
what if http.request.params.get("event") is not found ?

Reply:
@dbeguin An exception will be raised, but that means that someone forged an HTTP request